code/modules/power/gravitygenerator.dm

```diff
@@ -88,13 +88,17 @@
 /obj/machinery/gravity_generator/main/station/proc/round_startset()
 	if(round_start >= 1)
 		round_start--
+		set_light(8,1,LIGHT_COLOR_CYAN)
+
 //
 // Generator an admin can spawn
 //
 
 /obj/machinery/gravity_generator/main/station/admin/Initialize()
 	. = ..()
 	round_start = 1
+	set_light(8,1,LIGHT_COLOR_CYAN)
```

Comment:
Would like this converted to just setting the vars on the type.
Also, why only the `admin` type (whatever that is)?

Reply:
fiiixed?